apps/meteor/client/views/admin/rooms/RoomsTable.tsx

```diff
@@ -248,24 +233,24 @@ const RoomsTable = ({ reload }: { reload: MutableRefObject<() => void> }): React
 					</GenericTableBody>
 				</GenericTable>
 			)}
-			{isSuccess && data && data?.rooms.length > 0 && (
-				<>
-					<GenericTable>
-						<GenericTableHeader>{headers}</GenericTableHeader>
-						<GenericTableBody>{isSuccess && roomsList?.map((room) => renderRow(room))}</GenericTableBody>
-					</GenericTable>
-					<Pagination
-						divider
-						current={current}
-						itemsPerPage={itemsPerPage}
-						count={data?.total || 0}
-						onSetItemsPerPage={setItemsPerPage}
-						onSetCurrent={setCurrent}
-						{...paginationProps}
-					/>
-				</>
-			)}
-			{isSuccess && data && data.rooms.length === 0 && <GenericNoResults />}
+
```

Comment:
why? There's no need to show the table headers without results

Reply:
It already doesn't show it:
![image](https://github.com/RocketChat/Rocket.Chat/assets/43561537/9b1fc3b1-e97c-412c-bad0-9bc92c3b1e33)

I also found it weird that it did this but since it did I didn't botter adding the duplicated verification. Do you think its good to do it?